code/game/jobs/job/supervisor.dm

```diff
@@ -176,8 +176,13 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
 		H.equip_or_collect(new /obj/item/clothing/glasses/hud/health/health_advanced, slot_glasses)
 		H.equip_or_collect(new /obj/item/clothing/under/rank/blueshield(H), slot_w_uniform)
 		H.equip_or_collect(new /obj/item/clothing/suit/armor/vest/blueshield(H), slot_wear_suit)
-		H.equip_or_collect(new /obj/item/weapon/gun/energy/advtaser(H), slot_s_store)
 		H.equip_or_collect(new /obj/item/device/pda/heads/blueshield(H), slot_wear_pda)
+
+		if(H.backbag == 1)
+			H.equip_or_collect(new /obj/item/weapon/gun/energy/blueshield(H), slot_l_hand)
```

Comment:
@DaveTheHeadcrab 

This all need to be moved to down below, like @tigercat2000 stated.